OMF-Data.json

```diff
@@ -0,0 +1,40 @@
+[
+  {
+    "containerid": "Container1",
+    "values": [
+      {
+        "timestamp": null,
```

Comment:
any reason this is using different case than the other attributes?

Reply:
from PI to OCS this is Timestamp

Reply:
No particular reason for that. I just did not notice that it was not capitalized

Reply:
Having timestamp lowercase makes checking easier since that is what is returned by SDS